### ISSUE TITLE
Continuously Detected mining lock - 3080Ti dualmine ETH+TON 

### ISSUE THREAD

**Issue by kaitran1910** [Lolliedieb/lolMiner-releases#1388]
<img width="1244" alt="Screen Shot 2022-03-09 at 12 11 07 PM" src="https://user-images.githubusercontent.com/61256810/157494604-42064447-e653-4031-b4cb-9f182a81a65f.png">

Suddenly one day my 3080 Ti Ventus OC continuously get mining lock without any modification.
It's been like this for a couple days.
HiveOS newest version.

Here's my config for this 3080 Ti alone:
- Previous config (running well until this loop happening):
```
+ Locked core clock: 1400
+ Mem clock: 2600
+ --maxdualimpact: * (not specified)
+ --lhrtune: auto
```
ETH: 84 MH/s
TON: 2.4 GH/s

- Current config:
```
+ Locked core clock: 1600
+ Mem clock: 1800
+ --maxdualimpact: 2
+ --lhrtune: auto
```
ETH: --lhrtune keep going down  (first day when I saw it's -390 something), making ETH hashrate start decreasing from 76MH/s and getting worse
TON: fluctuating when LHR lock kicking in too, of course now it's about 2.7-3.0 GH/s since I increase Core Clock and --maxdualimpact to temporarily save the situation

Efforts tried:
- Changing OC: raising/lowering Locked Core Clock (from 1200 to 1600) and Mem Clock (from 1800-2600)
- Changing NVIDIA driver to 470.74 / 470.86 / 470.103, same results so I keep it 470.86. 
   Version 495.46 even makes things worse.
- Changing lolMiner version
- Changing riser, PSU
- Swapping Pcie slot on motherboard
- Swapping it to another rig with way cooler temperatures (48°-51° instead of 60° currently)
- Swapping it to my own Windows PC, running it alone at Pcie x16 slot
All of this efforts have failed. LHR lock keep kicking in and --lhrtune keep going down forever.

Other gpus on this rig still working fine
Other 3080Ti on different rig still working fine.

**Comment by nulledone** [comment]
Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.

**Comment by jgonzis** [comment]
> Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> 
> Edit: Driver version 511.79 lol.

For Windows the best version is 472.12 : https://github.com/Lolliedieb/lolMiner-releases/wiki/Guide-Windows-(LHR,Watchdog,WebServer)



> <img alt="Screen Shot 2022-03-09 at 12 11 07 PM" width="1244" src="https://user-images.githubusercontent.com/61256810/157494604-42064447-e653-4031-b4cb-9f182a81a65f.png">
> 
> Suddenly one day my 3080 Ti Ventus OC continuously get mining lock without any modification. It's been like this for a couple days. HiveOS newest version.
> 
> Here's my config for this 3080 Ti alone:
> 
> * Previous config (running well until this loop happening):
> 
> ```
> + Locked core clock: 1400
> + Mem clock: 2600
> + --maxdualimpact: * (not specified)
> + --lhrtune: auto
> ```
> 
> ETH: 84 MH/s TON: 2.4 GH/s
> 
> * Current config:
> 
> ```
> + Locked core clock: 1600
> + Mem clock: 1800
> + --maxdualimpact: 2
> + --lhrtune: auto
> ```
> 
> ETH: --lhrtune keep going down (first day when I saw it's -390 something), making ETH hashrate start decreasing from 76MH/s and getting worse TON: fluctuating when LHR lock kicking in too, of course now it's about 2.7-3.0 GH/s since I increase Core Clock and --maxdualimpact to temporarily save the situation
> 
> Efforts tried:
> 
> * Changing OC: raising/lowering Locked Core Clock (from 1200 to 1600) and Mem Clock (from 1800-2600)
> * Changing NVIDIA driver to 470.74 / 470.86 / 470.103, same results so I keep it 470.86.
>   Version 495.46 even makes things worse.
> * Changing lolMiner version
> * Changing riser, PSU
> * Swapping Pcie slot on motherboard
> * Swapping it to another rig with way cooler temperatures (48°-51° instead of 60° currently)
> * Swapping it to my own Windows PC, running it alone at Pcie x16 slot
>   All of this efforts have failed. LHR lock keep kicking in and --lhrtune keep going down forever.
> 
> Other gpus on this rig still working fine Other 3080Ti on different rig still working fine.

That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.

**Comment by nulledone** [comment]
> > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > Edit: Driver version 511.79 lol.
> 
> For Windows the best version is 472.12 : https://github.com/Lolliedieb/lolMiner-releases/wiki/Guide-Windows-(LHR,Watchdog,WebServer)
> 

Not for me though, I use various applications as well as gaming with another nvidia card on the rig. Also I got constant locks up until 511.79 for some reason don't know why.

> That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.

This card's stock thermal pads are the worst, I got 110' @ 59' core, changed it with 12.8 pads, my core went up (for a while after change it was 68' core 90' mem, now it is fine around 58-62) but memory is down to 84', but sadly at 95% fan.

**Comment by jgonzis** [comment]
> > > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > > Edit: Driver version 511.79 lol.
> > 
> > 
> > For Windows the best version is 472.12 : https://github.com/Lolliedieb/lolMiner-releases/wiki/Guide-Windows-(LHR,Watchdog,WebServer)
> 
> Not for me though, I use various applications as well as gaming with another nvidia card on the rig. Also I got constant locks up until 511.79 for some reason don't know why.
> 
> > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> 
> This card's stock thermal pads are the worst, I got 110' @ 59' core, changed it with 12.8 pads, my core went up (for a while after change it was 68' core 90' mem, now it is fine around 58-62) but memory is down to 84', but sadly at 95% fan.

Great to know, last Nvidia Drivers have got lot of problem with the LHR... they have push more lock, great you found a sweet configuration. Temp at 84C it is really good one... all under 94-96C it is acceptable for a GDDR6X memory.

**Comment by jgonzis** [comment]
> > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > Edit: Driver version 511.79 lol.
> 
> Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> 
> <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> 
> <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> 
> Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> 
> > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> 
> At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> 
> Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> 
> I will keep this thread updated.



> > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > Edit: Driver version 511.79 lol.
> 
> Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> 
> <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> 
> <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> 
> Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> 
> > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> 
> At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> 
> Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> 
> I will keep this thread updated.

Try to run this command and report the information:

**Comment by kaitran1910** [comment]
> > > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > > Edit: Driver version 511.79 lol.
> > 
> > 
> > Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> > <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> > <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> > Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> > > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> > 
> > 
> > At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> > Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> > I will keep this thread updated.
> 
> > > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > > Edit: Driver version 511.79 lol.
> > 
> > 
> > Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> > <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> > <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> > Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> > > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> > 
> > 
> > At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> > Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> > I will keep this thread updated.
> 
> Try to run this command and report the information: nvidia-smi -q -d PERFORMANCE | grep "SW Power Cap"
> 
> We will see if you have Thermal VRAM Problem

It really is having thermal problem.
While plugging it to my Windows PC, it's observable that MemTemp frequently jumping to 108°. 
The gpu just wanna save itself from the heat by constantly triggering the lock so that MemTemp can get down to about 90°, creating this infinite loop.

Even when I move the PC to the place that can cool down the core around 10-13°, MemTemp stay with the same problem.

I will close this thread since I gave up on it and swap it for another GPU.
Thanks for the discussion guys.

**Comment by jgonzis** [comment]
> > > > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > > > Edit: Driver version 511.79 lol.
> > > 
> > > 
> > > Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> > > <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> > > <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> > > Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> > > > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> > > 
> > > 
> > > At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> > > Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> > > I will keep this thread updated.
> > 
> > 
> > > > Had the same issue on Windows with exactly same card, lowered down the mclock to 1000 (2000 on hiveos I guess) and locking core to 1300 fixed it @lhrtune 4.0, currently at 82 / 2.3gh. My bet is that either lol is not optimized well for this card or the card is the shittiest 3080ti.
> > > > Edit: Driver version 511.79 lol.
> > > 
> > > 
> > > Earlier morning, I mistakenly read it as `--maxdualimpact 4.0` (haha idk why too) and changed it in my config (`Core: 1300, Mem:2000, maxdualimpact 4.0`). Surprisingly, the hashrate is now stable at 72 / 2.3 at `lhrtune -26` without continuously LHR lock kicking in since then, as you can see in my attached image of the lolMiner's log and the stats in HiveOS.
> > > <img alt="Screen Shot 2022-03-10 at 11 18 25 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709002-a525485d-81e5-428a-8e08-d9b87e6ad45e.png">
> > > <img alt="Screen Shot 2022-03-10 at 11 19 45 AM" width="1223" src="https://user-images.githubusercontent.com/61256810/157709005-363a2adc-849d-40eb-b07c-a114c501065c.png">
> > > Much appreciated. At least now I know there's an OC that can make my card stable. Now I will try to see whether changing to `--lhrtune 4.0` helps.
> > > > That could be also Memory VRAM Temp... a 60C in CORE that normally causes VRAM to be in 104 or more which trigger LHR and you will get continuos LHR Block. And more when you said that before was working OK... it could be before was 100-102C and now trigger that is the problem. Try to make a test only mininig with 3080TI with --devices 7 to avoid heat in the rig... that should confirm the problem.
> > > 
> > > 
> > > At first, I think so too. But as I mentioned before, bringing it to a rig that have cooler air (at about 48°-51° core) still have the same problem. Still, I just realize that maybe CoreTemp dropping about 10° doesn't mean MemTemp dropping at all. I completely forget checking this point when testing it with my PC
> > > Tonight, I will bring it back to my Windows PC to confirm the Memory VRAM Temp. I ordered a heatsink since I didn't really want to void the warranty. Will attach it to the card's backplate to see whether the situation is solved. If it's still locking, changing the thermal pad is inevitable then.
> > > I will keep this thread updated.
> > 
> > 
> > Try to run this command and report the information: nvidia-smi -q -d PERFORMANCE | grep "SW Power Cap"
> > We will see if you have Thermal VRAM Problem
> 
> It really is having thermal problem. While plugging it to my Windows PC, it's observable that MemTemp frequently jumping to 108°. The gpu just wanna save itself from the heat by constantly triggering the lock so that MemTemp can get down to about 90°, creating this infinite loop.
> 
> Even when I move the PC to the place that can cool down the core around 10-13°, MemTemp stay with the same problem.
> 
> I will close this thread since I gave up on it and swap it for another GPU. Thanks for the discussion guys.

Many thanks to you for testing and for the reply... it is the normal complaint miner problem 😅,
